ui/component/sideNavigation/view.jsx

```diff
@@ -219,9 +219,33 @@ function SideNavigation(props: Props) {
 
   const isPersonalized = !IS_WEB || isAuthenticated;
   const isAbsolute = isOnFilePage || isMediumScreen;
-  const microNavigation = !sidebarOpen || isMediumScreen;
+  const isMobile = useIsMobile();
 
-  const showSubscriptionSection = sidebarOpen && isPersonalized && subscriptions && subscriptions.length > 0;
+  const menuCanCloseCompletey = livestreamEnabled || isOnFilePage || isMobile;
```

Comment:
`livestreamEnabled` might not be correct.  It's leaving a blank gap in the front page.

![image](https://user-images.githubusercontent.com/64950861/145809077-805ffe5d-9a37-43ab-8377-c03f868fbfe2.png)

Reply:
You're right. I agree I don't think livestreamEnabled is correct.

I think I read that as "isOnTheLivestreamingPage" and wanted to add that in just to be sure I included that page.

It probably is being handled by "isOnFilePage" so I think it should be okay to remove livestreamEnabled.

I must have missed that because I don't think livestreaming is enabled on my account.

Reply:
removed livestreamEnabled from menuCanCloseCompletely